src/oscar/apps/order/abstract_models.py

```diff
@@ -223,24 +223,23 @@ def shipping_tax(self):
 
     @property
     def shipping_status(self):
-        events = self.shipping_events.all()
+        events = self.shipping_events.order_by('-date_created').all()
```

Comment:
Well, auto increment is a term from MySQL and refers to the practical default implementation of primary keys. A PostgreSQL sequence-based primary key exhibits the same characteristic. I don't know about oracle.

However a primary key may also be not autogenerated, i.e. a category slug. In that case it's not guaranteed to be created in order.

However what we really need to define here is which event we want in case that there are two with the same date_created. We could also define a unique constraint on date_created.